temi_code/curate_my_heart_counts_public_BRIDGE.R

```diff
@@ -3,183 +3,275 @@ library(tidyverse)
+  cardiovascular_satisfied_SchemaV3_v1 <- curate_table("syn4929530", "cardiovascular_satisfied_SchemaV3_v1")
+  
+  watchMotionActivityCollector_v1 <- curate_table("syn20563457", "watchMotionActivityCollector_v1")
+  
+  my_heart_counts <- bind_rows( #binds dataframes on top of one another, creating one HUGE dataframe
+    Six_Minute_Walk_Test_SchemaV4_v2, 
+    Six_Minute_Walk_Test_SchemaV4_v6,
+    ActivitySleep_v2, 
+    Adequacy_of_activity_mindset_measure_v1, 
+    Adequacy_of_activity_mindset_measure_v2,
+    Covid_19_recurrent_survey_v1, 
+    Covid_19_survey_v1,
+    Covid_19_survey_v2,
+    Default_Health_Data_Record_Table, 
+    Diet_survey_cardio_SchemaV2_v2, 
+    Exercise_process_mindset_measure_v1,
+    Exercise_process_mindset_measure_v2,
+    ExternalIdentifier_v1, 
+    HealthKitClinicalDocumentsCollector_v1, 
+    HealthKitClinicalRecordsCollector_v1, 
+    Heart_Rate_Recovery_v1, 
+    Heart_Rate_Training_v1, 
+    Illness_mindset_inventory_v1,
+    Illness_mindset_inventory_v2,
+    NonIdentifiableDemographicsTask_v3, 
+    Reconsent_v1, 
+    Resting_Heart_Rate_v1, 
+    Vaping_and_smoking_survey_v1,
+    Vaping_and_smoking_survey_v2,
+    Vaping_and_smoking_survey_v3,
+    cardiovascular_2_APHHeartAge_7259AC18_D711_47A6_ADBD_6CFCECDED1DF_v1,
+    cardiovascular_2_APHHeartAge_7259AC18_D711_47A6_ADBD_6CFCECDED1DF_v2,
+    cardiovascular_23andmeTask_v1, 
+    cardiovascular_6_Minute_Walk_Test_SchemaV4_v1, 
+    cardiovascular_6MWT_Displacement_Data_v1, 
+    cardiovascular_6MinuteWalkTest_v2, 
+    cardiovascular_ABTestResults_v1, 
+    cardiovascular_ActivitySleep_v1, 
+    cardiovascular_AwsClientIdTask_v1, 
+    cardiovascular_Diet_survey_cardio_v1, 
+    cardiovascular_Diet_survey_cardio_SchemaV2_v1, 
+    cardiovascular_HealthKitDataCollector_v1, 
+    cardiovascular_HealthKitSleepCollector_v1, 
+    cardiovascular_HealthKitWorkoutCollector_v1, 
+    cardiovascular_NonIdentifiableDemographics_v1,
+    cardiovascular_NonIdentifiableDemographicsTask_v2,
+    cardiovascular_appVersion, 
+    cardiovascular_daily_check_v1,
+    cardiovascular_daily_check_v2,
+    cardiovascular_day_one_v1, 
+    cardiovascular_displacement_v1, 
+    cardiovascular_heart_risk_and_age_v1, 
+    cardiovascular_motionActivityCollector_v1, 
+    cardiovascular_motionTracker_v1, 
+    cardiovascular_par_q_quiz_v1, 
+    cardiovascular_risk_factors_v1, 
+    cardiovascular_risk_factors_SchemaV2_v1, 
+    cardiovascular_satisfied_v1, 
+    cardiovascular_satisfied_SchemaV2_v1,
+    cardiovascular_satisfied_SchemaV3_v1,
+    watchMotionActivityCollector_v1) %>%
+    as_tibble() #changes dataframe into tibble format for cleaner look
+  print("finished bind rows")
   return(my_heart_counts)
-  print(my_heart_counts)
 }
 
+#adds columns to mhc df which show length of engagement by Week and Day intervals
 mutate_participant_week_day <- function(engagement) {
-  print("engagement")
+  print("mpwd running")
```

Comment:
```suggestion
  print("mutate_participant_week_day running")
```

Reply:
I recommend similar changes below.  Abbreviations are tempting and a guilty pleasure but for readability it is best to copy/paste the full function name